docs/modules/_filtering_redact_.md

```diff
@@ -54,7 +54,7 @@ ___
 
 ▸ **redact**(record: *[ISerializedHttp](../interfaces/_http_serializer_.iserializedhttp.md)*, properties: *`string`[]*, redactor?: *[Redactor](_filtering_redact_.md#redactor)*): [ISerializedHttp](../interfaces/_http_serializer_.iserializedhttp.md)
 
-Redact properties on the matching intercepted records. Note that header names are forced to lower case.
+Redact properties on the matching intercepted records. Note that header names are forced to lower case. Run redact after a request in spy mode to redact the specified properties in the generated mocks to save. Run redact before any requests in mock mode to redact the specified properties on all intercepted requests.
```

Comment:
This is a bit confusing to me. I have a suggestion here.

Split into two paragraphs. Leaving this as the first
```
Redact properties on the matching intercepted records. Note that header names are forced to lower case. Run redact after a request in spy mode to redact the specified properties in the generated mocks to save. 
```

For second paragraph I think spell out the use case of why you need to redact before. Idk the wording but something about how you may need to have future requests match the redacted mocks and in those cases you can run redact before...

```
When you want future requests to match against redacted